charts/hawkbit-update-server/values.yaml

```diff
@@ -96,6 +80,34 @@ configMap:
 spring:
   profiles: "mysql"
 config:
+  application:
```

Comment:
I miss the documentation for this section (at least a reference). Users would more likely accept the chart when they know what is meant with `config.application.hawkbit.dmf` (or similar) is meant.

Reply:
@BrandonSchmitt

Reply:
added the vars to the readme...